katas/content/multi_qubit_systems/Common.qs

```diff
@@ -0,0 +1,4 @@
+namespace Kata.MultiQubitSystems {
+    operation Foo() : Unit {
```

Comment:
Is this here because a common file is required?

Reply:
Yes, I'll remove it since it's not really used. I was just testing the infra.

Reply:
Removed.